architecture/evm/json_rpc_cache.go

```diff
@@ -290,6 +290,7 @@ func (c *EvmJsonRpcCache) Get(ctx context.Context, req *common.NormalizedRequest
 	return common.NewNormalizedResponse().
 		WithRequest(req).
 		WithFromCache(true).
+		WithEmptyBehavior(policy.EmptyState()).
```

Comment:
Do we really need to pass around the policy empty state via response? Can we just handle the requirement within Get() func itself?

e.g. if it says "empty response must be ignored" we just return a NotFound error...

Reply:
Separately, do we need to change anything in Set() func to respect this policy flag?

Reply:
seems like we already got it covered here? 

https://github.com/erpc/erpc/blob/d3a6d82d0763c2dd5fa62457cab709bae90980e3/architecture/evm/json_rpc_cache.go#L418